dsl_parser/tests/test_parser_api.py

```diff
@@ -2108,6 +2108,46 @@ def assert_operation(op, extra_properties=False):
         self.assertEqual(workflow2['parameters']['key']['default'], 'value')
         self.assertEqual(workflow['plugin'], constants.SCRIPT_PLUGIN_NAME)
 
+    def test_script_mapping_explicit_script(self):
```

Comment:
We should probably have a test that covers a single line inline script too?

Reply:
This test already has one! `op3: "ctx logger info 'hello world'"`

Reply:
So it does- nice!